jupyter_dashboards/nbextension/notebook/dashboard-view/dashboard-actions.js

```diff
@@ -22,6 +22,8 @@ define([
 ) {
     'use strict';
 
+    var NS = 'jupyter-dashboard';
+
```

Comment:
I agree with statement from morning scrum: don't make this "configurable", just change instances like in other files.